starters/tailwind/src/ComboBox.tsx

```diff
@@ -45,3 +46,7 @@ export function ComboBox<T extends object>(
 export function ComboBoxItem(props: ListBoxItemProps) {
   return <DropdownItem {...props} />;
 }
+
+export function ComboBoxSection<T extends object>(props: DropdownSectionProps<T>) {
```

Comment:
We fixed the combobox flickering in RSP. In RSP, after the fix, it has the behavior of going from a collection to no collection and flashes a dropdown with the height of one empty item. With RAC this empty collection shows with no height so it's more like a flicker. Since the behavior is consistent I don't think we will change it.